### PR TITLE
Code Review: Add docs for add-library scheme command (#677)

### PR DESCRIPTION
**Pull request by nthState**
Resolves: #677

### CODE REVIEW

File: docs/pages/app.md [hunk 1]
```diff
@@ -10,7 +10,13 @@ You can integrate Sketch into your app or workflow using the `sketch://` URL sch
 
 **Note:** If a document is already open the canvas view does not change.
 
-## Center layer and specify zoom level
+### Table of Contents
+
+- [Center layer and specify zoom level](#centerlayer)
+- [Run a plugin command](#runplugincommand)
+- [Add a Remote Library](#addremotelibrary)
```
Comment (christianklotz) [inline docs/pages/app.md]: Looking at the main Sketch documentation, maybe using the term shared Library is better overall

```suggestion
- [Add a shared Library](#add-shared-library)
```

File: docs/pages/app.md [hunk 2]
```diff
@@ -10,7 +10,13 @@ You can integrate Sketch into your app or workflow using the `sketch://` URL sch
...
+
+## Center layer and specify zoom level <a name="centerlayer"></a>
```
Comment (christianklotz) [inline docs/pages/app.md]:
```suggestion
## Center layer and specify zoom level <a name="center-layer"></a>
```

File: docs/pages/app.md [hunk 3]
```diff
@@ -36,7 +42,7 @@ document.selectedLayers.forEach(layer => console.log(layer.id))
 
 _Actual Size_ has a zoom level of `1`. To zoom out specify values between `0...1` while using values greater than `1` to zoom in, e.g. `2` for 200%.
 
-## Run a plugin command
+## Run a plugin command <a name="runplugincommand"></a>
```
Comment (christianklotz) [inline docs/pages/app.md]:
```suggestion
## Run a plugin command <a name="run-plugin-command"></a>
```

File: docs/pages/app.md [hunk 4]
```diff
@@ -87,3 +93,28 @@ function handleURL(context) {
 Open the following link to display a _Hello World_ message within an open document. Provide any URL encoded string with the `msg` query parameter.
 
 [`sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World`](sketch://plugin/com.example.sketch.messenger/message.show?msg=Hello%20World)
+
+## Add a Remote Library <a name="addremotelibrary"></a>
```
Comment (christianklotz) [inline docs/pages/app.md]:
```suggestion
## Add a shared Library <a name="add-shared-library"></a>
```

File: docs/pages/app.md [hunk 5]
```diff
@@ -87,3 +93,28 @@ function handleURL(context) {
...
+
+You can add a Remote Library to Sketch by providing a url to an appcast file.
```
Comment (christianklotz) [inline docs/pages/app.md]: I'd make it more explicit that it's about libraries hosted somewhere using RSS. Also for consistency with the other entries, please make sure to use identical formatting of variables and parameter names, plus put the main URL scheme command straight below the summary paragraph. (don't accept suggestion verbatim, indention of snippet isn't correct but required for the suggestion not to break).

```suggestion
Add a Library hosted on any server and shared using RSS by specifying the URL to the appcast.

    ```
    sketch://add-library?url=APPCAST_RSS_URL
    ```

To find out more about sharing your own libraries, see:

- [Sharing a Library](https://www.sketch.com/docs/libraries/#sharing-libraries) Sketch documentation
- [Appcast documentation](https://sparkle-project.org/documentation/)
```

File: docs/pages/app.md [hunk 6]
```diff
@@ -87,3 +93,28 @@ function handleURL(context) {
...
+
+### Parameters
```
Comment (christianklotz) [inline docs/pages/app.md]:
```suggestion
```

File: docs/pages/app.md [hunk 7]
```diff
@@ -87,3 +93,28 @@ function handleURL(context) {
...
+
+Provide an `encoded appcast url` parameter to the `add-library` command.
```
Comment (christianklotz) [inline docs/pages/app.md]: ### `url` string

URL-encoded string specifying the address of the appcast RSS file.

File: docs/pages/app.md [hunk 8]
```diff
@@ -87,3 +93,28 @@ function handleURL(context) {
...
+
+```sh
+sketch://add-library?url=AppCastURL
+```
+
+#### Example
+
+Open the link below to add a Remote Library to Sketch
```
Comment (christianklotz) [inline docs/pages/app.md]: Good example but it's not link, so instead describe _what_ it is:

```suggestion
URL to install the Apple iOS Sketch Library from [Apple Design Resources](https://developer.apple.com/design/resources/).
```

File: docs/pages/app.md [hunk 9]
```diff
@@ -87,3 +93,28 @@ function handleURL(context) {
...
+
+```sh
+sketch://add-library?url=https%3A%2F%2Fdeveloper.apple.com%2Fdesign%2Fdownloads%2Fsketch.rss
+```
+
+## Related resources
+
+- [Sharing a Library](https://www.sketch.com/docs/libraries/#sharing-libraries)
+- [Appcast documentation](https://sparkle-project.org/documentation/)
```
Comment (christianklotz) [inline docs/pages/app.md]: Let's drop this. Both links are very specific to the `add-library` command where both links are included now.

File: docs/pages/app.md (hunk 1 above)
Comment (nthState) [reply docs/pages/app.md]: Changed.

File: docs/pages/app.md (hunk 3 above)
Comment (nthState) [reply docs/pages/app.md]: Changed.

File: docs/pages/app.md (hunk 4 above)
Comment (nthState) [reply docs/pages/app.md]: Changed.

File: docs/pages/app.md (hunk 5 above)
Comment (nthState) [reply docs/pages/app.md]: Changed.

File: docs/pages/app.md (hunk 8 above)
Comment (nthState) [reply docs/pages/app.md]: Changed.

File: docs/pages/app.md (hunk 7 above)
Comment (nthState) [reply docs/pages/app.md]: Changed.

File: docs/pages/app.md (hunk 6 above)
Comment (nthState) [reply docs/pages/app.md]: Changed.

**Comment by nthState** [comment]
> Looking good now 👍 … just change branch as we want to merge documentation updates that are outside the Sketch release cycle directly into `master`.

Branch changed to master.

**Pull request merged**